src/Ninja/G1CP/Content/Misc/npc.d

```diff
@@ -219,3 +219,12 @@ func void G1CP_NpcIdSetAiVar(var int npcInstance, var string aiVarName, var int
     var C_Npc slf; slf = Hlp_GetNpc(npcInstance);
     G1CP_NpcSetAiVar(slf, aiVarName, value);
 };
+
+/*
+ * Rename an NPC
+ */
+funt int G1CP_NpcRename(var string npcName, var string originalStr, var string correctedStr) {
```

Comment:
Typo: funt -> func


Unlike the other recent PR, this function actually changes a particular property (the NPC's name) instead of an arbitrary string or integer. This makes this function much more reasonable than the others.

Nevertheless, as mentioned in #362, this extra function is not necessary unless there are more fixes of that kind.

Reply:
Thanks for the typo hint!

There is at least one other fix queued that will benefit from this refactor, so I think it's worth refactoring.